apps/web-giddh/src/assets/css/style-1.scss

```diff
@@ -736,15 +736,15 @@ aside-menu-account .container-fluid {
     text-transform: uppercase;
 }
 
-.txtLwc {
+.txt-lwc {
```

Comment:
if we update the naming convention of class so we need to update class as a generic name txt-lwcs not a generic name, update class name

Reply:
generic class name will be covered in code designing